python/opendht.pyx

```diff
@@ -57,6 +57,20 @@ cdef inline void shutdown_callback(void* user_data) with gil:
         cbs['shutdown']()
     ref.Py_DECREF(cbs)
 
+cdef inline bool get_filter(const cpp.Value& value, void *user_data) with gil:
+    cbs = <object>user_data
+    if 'filter' in cbs and cbs['filter']:
+        # Deep of const Value to Python level Value
```

Comment:
oh god.
I see why you did that, but it's horribly inefficient. Maybe filters could always be null when using the Python bindings.
The "filter" feature is actually not that useful: users can always filter the received values in the get callback, it's much more efficient in this case, and will likely make user code cleaner and more readable, since it's a native Python API:
https://docs.python.org/3/library/functions.html#filter